generator_process/__init__.py

```diff
@@ -302,8 +317,9 @@ def main():
             # Couldn't track down exactly where it was hanging but it's good enough.
             from skimage import transform
```

Comment:
Forgot to previously test with only dream studio dependencies. Scikit-image is not a part of them.
```suggestion
            if os.path.exists(absolute_path(".python_dependencies/skimage")):
                from skimage import transform
```